### ISSUE TITLE
The rules "ter-no-proto", "ter-no-script-url", "ter-no-self-compare" are not available

### ISSUE THREAD

**Issue by vyshkant** [buzinas/tslint-eslint-rules#307]
### TSLint version:
```bash
$ tslint --version
5.9.1
```

### TypeScript version:
```bash
$ tsc --version
Version 2.7.2
```

### tslint.json configuration:
```json
{
  "extends": [
    "tslint-eslint-rules"
  ],
  "rules": {
    "ter-no-proto": [true],
    "ter-no-script-url": [true],
    "ter-no-self-compare": [true]
  }
}
```

### Actual behavior:
```bash
Could not find implementations for the following rules specified in the configuration:
    ter-no-proto
    ter-no-script-url
    ter-no-self-compare
```

### Expected behavior:
No errors, because there are the docs pages describing usage of these rules:
- https://github.com/buzinas/tslint-eslint-rules/blob/master/src/docs/rules/terNoProtoRule.md
- https://github.com/buzinas/tslint-eslint-rules/blob/master/src/docs/rules/terNoScriptUrlRule.md
- https://github.com/buzinas/tslint-eslint-rules/blob/master/src/docs/rules/terNoSelfCompareRule.md

**Comment by vyshkant** [comment]
So we need either to delete the docs page, or implement the rule :)

**Comment by buzinas** [comment]
> Maybe these rules are not released yet?

Exactly. I'll release next version on Friday.

**Comment by vyshkant** [comment]
> Exactly. I'll release next version on Friday.

@buzinas What Friday did you mean? :smile:
